model/qti/response/SimpleFeedbackRuleScore.php

```
@@ -0,0 +1,43 @@
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2025 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoQtiItem\model\qti\response;
+
+class SimpleFeedbackRuleScore extends SimpleFeedbackRule
+{
+    private string $xml;
+    public function toArray($filterVariableContent = false, &$filtered = []): array
```

PHPMD (2.15.0)

The method toArray has a boolean flag argument $filterVariableContent, which is a certain sign of a Single Responsibility Principle violation. (Clean Code Rules)

(BooleanArgumentFlag)